test/unit/models/noise_funnel_test.rb

```diff
@@ -0,0 +1,74 @@
+require "test_helper"
+
+class NoiseFunnelTest < ActiveSupport::TestCase
+  setup do
+    @potrero_hill_funnel = NoiseFunnel.new(
+      Neighborhood.districts[:potrero_hill].latitude,
+      Neighborhood.districts[:potrero_hill].longitude,
+      1.5,
+      0.025,
+      Time.current,
+      1.week.ago,
+    )
+  end
+
+  test '#noises includes nearby noises that were just created' do
+    noise = FactoryGirl.create(
+      :noise,
+      :potrero_hill,
+      created_at: Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.